docs/pages.jl

```diff
@@ -1,15 +1,24 @@
 pages = Any["Home" => "index.md",
-            "Catalyst for new Julia users" => Any["catalyst_for_new_julia_users/intro_to_catalyst.md"],
-            "Tutorials" => Any["tutorials/using_catalyst.md",
-                               "tutorials/dsl.md",
-                               "tutorials/reaction_systems.md",
-                               "tutorials/basic_examples.md",
-                               "tutorials/compositional_modeling.md",
-                               "tutorials/symbolic_stoich.md",
-                               "tutorials/reaction_network_representation.md",
-                               "tutorials/homotopy_continuation_tutorial.md",
-                               "tutorials/bifurcation_diagram.md",
-                               "tutorials/parameter_estimation.md",
-                               "tutorials/generating_reactions_programmatically.md"],
+            "Introduction to Catalyst" => Any[
+                                "intro_to_catalyst/catalyst_for_new_julia_users.md",
+                                "intro_to_catalyst/using_catalyst.md"
+                            ],
+            "Catalyst Functionality" => Any[
+                                "catalyst_functionality/dsl.md",
+                                "catalyst_functionality/programmatic_CRN_construction.md",
+                                "catalyst_functionality/basic_CRN_examples.md",
+                                "catalyst_functionality/compositional_modeling.md",
+                                "catalyst_functionality/parametric_stoichiometry.md"
+                            ],
+            "Tutorials" => Any[
+                                "tutorials/network_analysis.md",
```

Comment:
I think this should be in "Catalyst Functionality" since that is what it provides. 

Maybe call "Tutorials" something like "Advanced Tutorials" or "Catalyst Applications"? Just so users don't think it is beginner material?